CLogger/include/Utils/Utils.h

```diff
@@ -6,16 +6,18 @@
 /// </summary>
 namespace LogUtils
 {
+	CLOGGER_API std::string GetCurrentTimeString();
```

Comment:
It would be better if these functions reduce copying strings(I mean return string by ref). I saw that you use them in macros.

Reply:
OK, it will be fixed.

Reply:
Stop, but what about <a href="https://docs.microsoft.com/ru-ru/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4172?f1url=%3FappId%3DDev16IDEF1%26l%3DRU-RU%26k%3Dk(C4172)%26rd%3Dtrue&view=msvc-160">C4172</a>?

Reply:
I mean std::string GetCurrentTimeString(); -> void/bool GetCurrentTimeString(std::string &); but your macros should be reworked then